Rational Numbers exercise test is failing on CI

In #254 CI was failing for an unrelated reason.

<details>
 <summary>Log of failure on CI</summary>

```
$ tsc --noEmit -p . && jest --no-cache
(node:11406) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
 FAIL  ./rational-numbers.test.ts
  Addition
    ✓ Add two positive rational numbers (3ms)
    ✓ Add a positive rational number and a negative rational number (1ms)
    ✓ Add two negative rational numbers
    ✓ Add a rational number to its additive inverse
  Subtraction
    ✓ Subtract two positive rational numbers
    ✓ Subtract a positive rational number and a negative rational number (1ms)
    ✓ Subtract two negative rational numbers
    ✓ Subtract a rational number from itself
  Multiplication
    ✓ Multiply two positive rational numbers
    ✓ Multiply a negative rational number by a positive rational number
    ✓ Multiply two negative rational numbers (1ms)
    ✓ Multiply a rational number by its reciprocal
    ✓ Multiply a rational number by 1
    ✓ Multiply a rational number by 0
  Division
    ✓ Divide two positive rational numbers
    ✓ Divide a positive rational number by a negative rational number
    ✓ Divide two negative rational numbers (1ms)
    ✓ Divide a rational number by 1
  Absolute value
    ✓ Absolute value of a positive rational number
    ✓ Absolute value of a negative rational number
    ✓ Absolute value of zero
  Exponentiation of a rational number
    ✓ Raise a positive rational number to a positive integer power (1ms)
    ✓ Raise a negative rational number to a positive integer power
    ✓ Raise zero to an integer power
    ✓ Raise one to an integer power
    ✓ Raise a positive rational number to the power of zero
    ✓ Raise a negative rational number to the power of zero
  Exponentiation of a real number to a rational number
    ✓ Raise a real number to a positive rational number (1ms)
    ✕ Raise a real number to a negative rational number (169ms)
    ✓ Raise a real number to a zero rational number
  Reduction to lowest terms
    ✓ Reduce a positive rational number to lowest terms
    ✓ Reduce a negative rational number to lowest terms (1ms)
    ✓ Reduce a rational number with a negative denominator to lowest terms
    ✓ Reduce zero to lowest terms
    ✓ Reduce an integer to lowest terms
    ✓ Reduce one to lowest terms
  ● Exponentiation of a real number to a rational number › Raise a real number to a negative rational number
    expect(received).toEqual(expected)
    
    Expected value to equal:
      0.3333333333333333
    Received:
      0.33333333333333337
      
      at Object.<anonymous> (rational-numbers.test.ts:158:48)
          at new Promise (<anonymous>)
Test Suites: 1 failed, 1 total
Tests:       1 failed, 35 passed, 36 total
Snapshots:   0 total
Time:        1.605s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make[1]: *** [test-assignment] Error 1
make[1]: Leaving directory `/home/travis/build/exercism/typescript'
make: *** [test] Error 1
The command "make test" exited with 2.
```

</details>

--- a/exercises/rational-numbers/rational-numbers.test.ts
+++ b/exercises/rational-numbers/rational-numbers.test.ts
@@ -154,7 +154,7 @@ describe('Exponentiation of a real number to a rational number', () => {
     })
 
     xit('Raise a real number to a negative rational number', () => {
-        const expected = 0.3333333333333333
+        const expected = 1.0/3.0
         expect(new Rational(-1, 2).expreal(9)).toEqual(expected)
     })
 

